include/zephyr/dt-bindings/pwm/stm32_pwm.h

```diff
@@ -16,10 +16,10 @@
  * @{
  */
 /** PWM complementary output pin is enabled */
-#define PWM_STM32_COMPLEMENTARY	(1U << 8)
+#define STM32_PWM_COMPLEMENTARY	(1U << 8)
```

Comment:
Let's make change non breaking
```suggestion
#define PWM_STM32_COMPLEMENTARY	__DEPRECATED_MACRO (1U << 8)
#define STM32_PWM_COMPLEMENTARY	(1U << 8)
```

Reply:
ok